src/utils/category.js

```diff
@@ -0,0 +1,23 @@
+export const categoryItems = (products, workout, tennis, running) => {
+  console.log("category input", products);
```

Comment:
You can remove all the console logs when you create a PR.

Reply:
done